CenterForOpenScience/ember-osf-web
# [ENG-4419] No BsDropdown

-   Ticket: [ENG-4419]
-   Feature flag: n/a

## Purpose

Remove usage of BsDropdown from the ember-osf-web codebase.

## Summary of Changes

Remove BsDropdown from:

1. Analytics page
2. Node card
3. Collections Discover page
4. OSF Navbar and OSF Navbar Auth Dropdown
5. Registries Navbar

## Screenshot(s)

Analytics page:
<img width="243" alt="Screenshot 2023-05-02 at 9 38 15 AM" src="https://user-images.githubusercontent.com/6599111/236227024-e856e9c7-0893-4d3b-a51a-0bf43e72bc1d.png">

Node Card:
<img width="793" alt="Screenshot 2023-05-02 at 10 15 51 AM" src="https://user-images.githubusercontent.com/6599111/236227268-5261c5fd-2816-41a6-941c-8a1aa5379c5d.png">

Collections Discovery page:
<img width="701" alt="Screenshot 2023-05-02 at 10 48 06 AM" src="https://user-images.githubusercontent.com/6599111/236227442-41a6f16e-5f64-4957-869b-28b785be3625.png">

OSF Navbar:
<img width="258" alt="Screenshot 2023-05-02 at 2 32 35 PM" src="https://user-images.githubusercontent.com/6599111/236227599-464a9fe4-1e73-464e-b692-f854f7dbede2.png">

<img width="242" alt="Screenshot 2023-05-02 at 3 09 57 PM" src="https://user-images.githubusercontent.com/6599111/236227705-64936fd1-db5c-44c7-8811-7e598872a1fd.png">

Registries Navbar:
<img width="331" alt="Screenshot 2023-05-04 at 8 55 23 AM" src="https://user-images.githubusercontent.com/6599111/236227845-07f5284d-eb8d-42fe-90e1-e942d3db765c.png">
<img width="254" alt="Screenshot 2023-05-04 at 9 00 49 AM" src="https://user-images.githubusercontent.com/6599111/236227864-b9ead882-dd8e-4b0d-9909-1a43e55691ec.png">



## Side Effects

The look may be a bit different in places

## QA Notes

Just make sure the menus still work and don't look weird. For navbars, be sure to check the logged-out state as well.


[ENG-4419]: https://openscience.atlassian.net/browse/ENG-4419?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
